.gitignore

```diff
@@ -1,6 +1,9 @@
 target/
 !.mvn/wrapper/maven-wrapper.jar
 
+node_modules
+package-lock.json
```

Comment:
@ARKEOLOGIST let's add package-lock.json
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5